test/EventStore.Client.IntegrationTests/EventStoreClientFixture.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Ductus.FluentDocker.Builders;
+using Ductus.FluentDocker.Services;
+using EventStore.Client.Gossip;
+using Grpc.Core;
+#if !GRPC_CORE
+using Grpc.Net.Client;
+#endif
+using Polly;
+using Xunit;
+using EndPoint = System.Net.EndPoint;
+
+#nullable enable
+namespace EventStore.Client {
+	public class Somerting {
+		[Fact]
+		public async Task YouSuck() {
+			await using var fixture = new EventStoreClientInsecureClusterFixture();
+			await fixture.Start();
+			using var client = fixture.CreateEventStoreClient(
+				Array.ConvertAll(fixture.Ports, port => new IPEndPoint(IPAddress.Loopback, port)));
+
+			var leader = await fixture.GetLeaderNode();
+
+			leader!.ToString();
+		}
+	}
```

Comment:
@thefringeninja this looks like some code left over by mistake ;)

Reply:
Derp! Removed